modules/log4cats/src/main/scala/doobie/log4cats/Log4CatsLogHandler.scala

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2013-2020 Rob Norris and Contributors
+// This software is licensed under the MIT License (MIT).
+// For more information see LICENSE or https://opensource.org/licenses/MIT
+
+package doobie.log4cats
+
+import doobie.LogHandler
+import doobie.util.log._
+import org.typelevel.log4cats._
+
+class Log4CatsLogHandler[F[_]](logger: MessageLogger[F]) extends LogHandler[F]{
+  override def run(logEvent: LogEvent): F[Unit] = logEvent match {
+    case Success(s, a, l, e1, e2) =>
+      logger.info(
+        s"""Successful Statement Execution:
+           |
+           |  ${s.linesIterator.dropWhile(_.trim.isEmpty).mkString("\n  ")}
+           |
+           | arguments = [${a.mkString(", ")}]
```

Comment:
I think logging arguments by default would be a security violation in many workplaces. Perhaps we can make this an option (default false)

Reply:
I just copied whatever `jdkLogHandler` does.
https://github.com/tpolecat/doobie/blob/main/modules/free/src/main/scala/doobie/util/log.scala#L66

Reply:
jdkLogHandler is not for production use (as written in the scaladoc). Although I think we can update that scaladoc too to provide additional info

Reply:
This class would still be very valuable for debugging purposes in non-production environments.
Indeed it is true that it could log sensitive data.